Phonebook.Frontend/src/app/pages/page-information/page-information.component.scss

```diff
@@ -1,3 +1,28 @@
-.small-card {
-  width: 50%;
+.mat-card{
```

Comment:
same for this

Reply:
Also for the future, please do not reuse classes from the angular framework. Once it is in the code and you want to debug where the style is coming from you are determined to fail as you look up why the heck this class is behaving not like described in the documentation of the framework. ;)